### ISSUE TITLE
Wrong og:url on Multilingual Content (Using Polylang)

### ISSUE THREAD

**Issue by black-eye** [sybrew/the-seo-framework#392]
Hi,

see page here: https://shaktiqueen.blackeye.cz/en/
The og:url points out to https://shaktiqueen.blackeye.cz and thus (propably) when sharing the page on FB it shows the wrong language (default one).

Just for the reference, the same problem isn't present neither in Yoast (http://demo.remedical.cz/en/) nor in SEOPress (https://www.playday.cz/en/), so I suppose it's a bug, not an intention.

Thanks

**Comment by black-eye** [comment]
Hm, just an addiction: it seems, that the problem is specific for the Home page only, subpages (e.g. https://shaktiqueen.blackeye.cz/en/birthing-the-goddess/) are fine.

**Comment by black-eye** [comment]
@sybrew Thanks for the tip, the canonical URL in SEO Settings did the trick. 

Strangely enough, the right address (/en) was there prefilled as a placeholder, but only after filling it regularly, it took the effect on the frontend.

**Comment by sybrew** [comment]
Yes, that's part of their monkey patch 😄, this is a good sign, thanks for highlighting it! 

I'll see if I can send Polylang a PR that fixes this in `wp_head` (where TSF outputs its metadata), rather than in the body and on the admin pages.

**Comment by MikeiLL** [comment]
I don't see which setting is being referred to on the `/wp-admin/options-reading.php` page:

> The only solution would be to fill in a custom canonical URL for the translated homepage, but it has to be a page, not a blog. This value can be set at `/wp-admin/options-reading.php`. 

However I did, "set the canonical URL when editing the page itself, within the SEO Settings meta box at the bottom."

When the URL was set to `https://example.com/es/page-title` it created a redirect loop:

Input URL | arrow-right | https://nydems.org/es/la-portada/
og:url Meta Tag | arrow-right | https://nydems.org/es/la-portada
301 HTTP Redirect | arrow-right | https://nydems.org/es/la-portada/
og:url Meta Tag | arrow-right | https://nydems.org/es/la-portada

This is because I think Polylang creates a redirect. So the Cannonical URL needs to be like: ``https://example.com/page-title`. Now the correct `og` tags are being read. As is the correct title.
